src/services/User.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import usersFromServer from '../api/users';
+
+export function getUserId(userId: number): User | null {
```

Comment:
The function name `getUserId` is misleading because it returns a `User` object, not just an ID. Consider renaming it to something like `getUserById` to better reflect its functionality.